specs/PermissionManagement.md

```diff
@@ -379,6 +380,12 @@ typedef enum COREWEBVIEW2_PERMISSION_KIND {
   /// developers use the [Local Font Access API](https://wicg.github.io/local-font-access/)
   /// to query the system fonts available for styling web content.
   COREWEBVIEW2_PERMISSION_KIND_LOCAL_FONTS,
+
+  /// Indicates permission for a site to use system exlusive messages to access
```

Comment:
Wording is inconsistent on this new entry, "Indicates permission for a site to [do something]" vs "Indicates permission to [do something]".  Does the "for a site" phrase indicate something unique about this PermissionKind enum?  (I don't think so, I think the "a site" is just implicit in all these enums.)   Be consistent unless there's a reason not to be.

Reply:
Please update. We can be consistent here and remove "for a site". The context in which the permission kind enum value is used makes it clear this is for a site rather than this enum itself.